assertion_test.go

```diff
@@ -477,3 +477,183 @@ func TestAssertion_Strings(t *testing.T) {
 		}
 	})
 }
+
+func TestAssertion_ValidateTraits(t *testing.T) {
+	tests := []struct {
+		testName          string
+		errorContainsText string
+		failure           AssertionFailure
+		traits            fieldTraits
+	}{
+		{
+			testName:          "bad Type",
+			errorContainsText: "AssertionType",
+			failure: AssertionFailure{
+				Type: AssertionType(9999),
+			},
+			traits: fieldTraits{
+				Actual: fieldRequired,
+			},
+		},
+		{
+			testName:          "required Actual",
+			errorContainsText: "should have Actual",
```

Comment:
Here and below, let's relax requirements for error messages as much as possible. Here I suggest to just check that the error message contains "Actual".

Otherwise we'll have to fix all tests whenever we change wording of errors.